frontend/src/styles/vars.scss

```diff
@@ -316,7 +314,6 @@ $_lifecycle_dormant: map.get($colors, 'danger');
 
 // defined here so that they can be shared with the toolbar
 @mixin dark-mode-3000-variables {
-    --default: var(--default-dark);
```

Comment:
This wasn't right @daibhin! `default-dark` has meant a darker take on `default`, not the dark-_mode_ variant of it. Fixed by making sure `--default` always is just an alias of `text-3000`.

Reply:
Ah, apologies! Certainly a little confusing - any reason we couldn't remove the references to `-default` and just use `-text-3000` instead?